src/item.c

```diff
@@ -48,7 +48,12 @@ item_new (void)
 itemPtr
 item_load (gulong id)
 {
-	return db_item_load (id);
```

Comment:
While the modification of the item loading interface is functionally valid I still think we should do this UI thingy only in the UI code in item_list_view_update_item_internal(). I think moving it there might even save as from a lot of extra string formattings that do never touch the UI (think search folder rebuilds, our item list merging on feed updates).

Reply:
@rich-coe What do you think?

Reply:
i'll take a look.

Reply:
@lwindolf  adding timestr was an optimization to allocate and format the date string only once during the item fetch from the db, instead of every time the item gets rendered into the UI.

I've modifed it to only format the date string in item_list_view_update_item_internal().